src/components/breadcrumb/BreadcrumbItem.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+    <li
+        :class="{ 'is-active': active }">
+        <component
+            :is="tag"
+            :href="tag == 'a' ? href : undefined"
```

Comment:
Please copy it https://github.com/buefy/buefy/blob/dev/src/components/navbar/NavbarItem.vue

There is no need to add href and to props